### ISSUE TITLE
Scss compilation fails due to fancy quotes.

### ISSUE THREAD

**Issue by kicken** [zurb/foundation-sites#7210]
#7131 introduced fancy quotes some comments, which causes compilation to fail using the ruby sass gem on windows.

	Error: Invalid Windows-1252 character "\xE2"
		on line 76 of D:/Website/bower_components/foundation-sites/scss/util/_breakpoint.scss
		from line 12 of D:/Website/bower_components/foundation-sites/scss/util/_util.scss
		from line 9 of D:/Website/bower_components/foundation-sites/scss/foundation.scss
		from line 6 of web/src/scss/main.scss
	  Use --trace for backtrace.

**Comment by ivanjaros** [comment]
Maybe you have edited the file in your editor with wrong charset? I have no issue on windows.10(using gulp, nodejs, bower, phpstorm, git for windows).

**Comment by kicken** [comment]
I had considered that already.  Prior to posting I did a few tests including clearing my bower_components and re-installing and also doing a direct git clone.  Each produced the error.

I did a little more testing today and it appears it's related to bower.  Bower ends up changing the console code page during it's install command which causes the issue.  Opening a fresh console after `bower install` or resetting the code page with `mode con: cp select=437` resolves the problem.

These additional steps were not necessary prior to the referenced commit.
